src/channeldef.ts

```diff
@@ -943,11 +943,11 @@ export function title(
   }
 }
 
-export function getGuide(fieldDef: TypedFieldDef<string> | SecondaryFieldDef<string> | DatumDef): Guide {
+export function getGuide(fieldDef: TypedFieldDef<string> | SecondaryFieldDef<string> | DatumDef): Guide<SignalRef> {
```

Comment:
`getGuide` is called in general utility function, not just inside init(). 

How do we know that this is `Guide<SignalRef>`, not `Guide<Expr>`?

Reply:
I guess if we call this function after init, the guide should only have Expr refs, no?